Documentation/HowTo/add-syntax-highlighting-with-pygments.md

```diff
@@ -0,0 +1,64 @@
+# How to add syntax highlighting with Pygments
+[Splash](https://github.com/JohnSundell/Splash) and its [official plugin](https://github.com/JohnSundell/SplashPublishPlugin) are great tools for highlighting swift syntax in Publish. 
+
+However, some people write not only in Swift, but also in many other languages. There comes the [SwiftPygmentsPublishPlugin](https://github.com/Ze0nC/SwiftPygmentsPublishPlugin) to help. 
+
+## [SwiftPygmentsPublishPlugin](https://github.com/Ze0nC/SwiftPygmentsPublishPlugin)
+
+It is a Pygments plugin for [Publish](https://github.com/johnsundell/publish) to highlight code. 
+[Pygments](https://pygments.org) is a syntax highlighting tool made in Python that supports many languages. 
+
+## Requirements
```

Comment:
Rather than putting instructions inline here, I'd prefer for this and the following "How to" section to simply reference your plugin's README. Otherwise, I'll have to make sure to keep this up to date as you change your implementation and requirements, which'll most likely result in this becoming out of date. Take a look at [the Splash plugin's "How to" guide for example](https://github.com/JohnSundell/Publish/blob/master/Documentation/HowTo/adding-swift-syntax-highlighting.md). I think this file should only contain step number 4, that is, how to insert your plugin into a publishing pipeline.

Reply:
I have moved the detailed guide to plugin's README.md and simplified the How-to ariticle.